app/models/endpoint.rb

```diff
@@ -11,4 +11,24 @@ class Endpoint < ApplicationRecord
   validates :endpoint_node, presence: true
   validates :storage_location, presence: true
   validates :recovery_cost, presence: true
+
+  # rubocop:disable Metrics/AbcSize
+  # TODO: find a clean way to satisfy this style check, everything that silenced the complaint for me was
+  # less readable to my eyes due to what felt like needless indirection.
+  def self.seed_local_online_endpoints_from_config(preservation_policies = nil)
```

Comment:
decide to use "default" or "local_online" and use it consistently in the method name and var names  (or "default_local_online" or whatever).  I keep having to check what we're talking about.

Reply:
well, i don't know that the names should line up so nicely, since they aren't referring to quite the same thing...

i don't really think we have the concept of a "default" endpoint or a "default" endpoint type in general (right?).  so it doesn't really make sense to me to name the `EndpointType` method `EndpointType.default_endpoint_type`.  there's a default configuration for local online endpoints, but not endpoint types in general.  so that's what `EndpointType.default_local_online_endpoint_type` gets.

similarly, with `Endpoint.seed_local_online_endpoints_from_config`...  i could've just done `Endpoint.seed_from_config`, as with the other classes, but i guess i wanted to make it clear that the method only does local endpoints at the moment, since there will be other endpoint types that need configuring as we do more development.  and since there didn't seem to me to be a concept of "default" endpoints, `.seed_default_endpoints_from_config` felt wrong.  though i'm not sure i made the right choice, and would totally be open to just calling it `Endpoint.seed_from_config`, with the assumption that the underlying code will be refactored, and `seed_from_config` will do all the seeding, for all types of endpoints, as we continue to develop them.

i dunno... `Endpoint.seed_local_online_endpoints_from_config` seeds local online endpoints from configuration, using the default configuration for online local endpoints, obtained via `EndpointType.default_local_online_endpoint_type`.  i know this is verbose, and the names seem like they don't quite fit the same pattern, but i'd find `Endpoint.seed_default_endpoints_from_config` (or any version of that method name with `default` in it) to be misleading.

um... how about `EndpointType.default_local_online_endpoint_type` and `Endpoint.seed_from_config`?  less verbose, less seeming mismatch in terminology, still entirely truthful about what its doing (assuming my understanding of endpoints and endpoint types is correct).

Reply:
if you can grab tommy before he leaves and agree on terms, go for it.  I may have confused the "default" prez. policy with the endpoint type stuff (which is telling in my ability to read the code).   I think i am also slightly confused by "default configuration for local online endpoints" - is it likely to change, except for the diff between local dev (with fixtures) and deployed?   

Which configuration data in settings expects certain things first?  that feels "default"-y and I agree with tommy - it could be clearer to call it out explicitly in the config as default rather than making it position dependent.

basically, my overall sentiments are:
- the code in this method, as written, wasn't skimmably clear to me -- as someone less intimately familiar with the database 
- use explicit default rather than positional if simple to do
- oy, super long names.
- eventually, we'll seed all the endpoints from config, no?  not just online local?   how might that impact naming choices?